taxa/taxonomy/taxonomydynamicdisplay.php

```diff
@@ -115,6 +115,7 @@ function displayTaxomonyMeta(){
 	</div>
 	<!-- This is inner text! -->
 	<div id="innertext">
+		<h1 class="page-heading"><?php echo 'Taxonomy Explorer: '.$taxonDisplayObj->getTargetStr(); ?></h1>
```

Comment:
Change to "Taxonomy Explorer: Central Thesaurus"
![image](https://github.com/BioKIC/Symbiota/assets/16688784/a0376329-b632-452f-a337-d22e0dec86af)
and remove the other Central Thesaurus

Reply:
Fixed in 9e6ef7ea3.